### DISCUSSION TITLE
Unable to do pip install sqlalchemy

### DISCUSSION THREAD

**Discussion by Lainelainex3** [sqlalchemy/sqlalchemy#10796] [Usage Questions]
Error when I try to do pip install sqlalchemy in ubuntu: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/sqlalchemy/

**Comment by CaselIT** [comment]
hi,

that seems to be a pypi issue. I don't see how the sqlalchemy maintainers can do anything about it

**Reply by Lainelainex3** [reply]
What is the solution? How do I install sqlalchemy?

**Reply by CaselIT** [reply]
In the classical way you do when a website does not work.

The easiest thing is to wait for pypi to fix their issue. Otherwise the github releases include wheels you can download and install https://github.com/sqlalchemy/sqlalchemy/releases

**Comment by Lainelainex3** [comment]
lol i'm able to install sqlalchemy today. ig the pypi has connectivity issue yesterday.
